typescript-react/src/components/View/helpers.ts

```diff
@@ -70,7 +70,6 @@ export const getResultSetColumnDefinitions = (
         case ColumnType.Float:
         case ColumnType.Int:
         case ColumnType.Long:
-        case ColumnType.Short:
```

Comment:
but short are numbers. you should handle this somewhat differently?

Reply:
Would it make more sense that it returns it's own `ColumnType` (like `Double` in this example) and that the `ColumnType.Short` type uses the `TextCell` component?